### DISCUSSION TITLE
Sepolia Testnet Faucet

### DISCUSSION THREAD

**Discussion by RSKKSOFFICIAL** [smartcontractkit/full-blockchain-solidity-course-py#1911] [General]
I need some eth faucet in sepolia testnet, please help me with that.

0x868d6fe4E50cAf004944CdD7b06874e4c7333D66

**Comment by SANTHSIM22** [comment]
@RSKKSOFFICIAL  use this https://cloud.google.com/application/web3/faucet/ethereum/sepolia
